src/NftDetails/NftDetailsImage.tsx

```diff
@@ -1,30 +1,37 @@
-import {useRoute} from '@react-navigation/native'
 import React from 'react'
-import {Image, StyleSheet, View} from 'react-native'
+import {Dimensions, Image, StyleSheet, View} from 'react-native'
+import ViewTransformer from 'react-native-easy-view-transformer'
 
 import {FadeIn, OfflineBanner, StatusBar} from '../components'
 import {useNfts} from '../hooks'
 import {useSelectedWallet} from '../SelectedWallet'
 
-type Params = {id: string}
+type Props = {route: {params: {id: string}}}
 
-export const NftDetailsImage = () => {
+export const NftDetailsImage = ({
+  route: {
+    params: {id},
+  },
+}: Props) => {
   const wallet = useSelectedWallet()
   const {nfts} = useNfts(wallet)
 
-  const {id} = useRoute().params as Params
   const nft = nfts.find((nft) => nft.id === id)
 
-  if (!nft) {
+  if (nft === undefined) {
     return null
   }
+  const dimensions = Dimensions.get('window')
+  const imageSize = Math.min(dimensions.width, dimensions.height)
 
   return (
     <FadeIn style={styles.container}>
       <StatusBar type="dark" />
       <OfflineBanner />
```

Comment:
do we need it? `.image` is sync available I don't think there's a reason to display the offline banner, p.s haven't checked figma